docs/design-docs/multi-DM.md

```diff
@@ -66,3 +70,57 @@ behaviors, namely:
      LockingTransactions.client - to ensure that the lock identifier
      is consistent across all replicas.  This change should be
      straightforward.
+     
+## AtLeastOnceRPC + DHT + ConsensusRSM
+
+### Desired behavior
+
+1. Client creates a new instance or obtains a reference to an existing MicroService.
+2. Client makes a call to an application method in Microservice and gets the result back.
+3. Client DM retries on failure for certain duration.
+4. Microservice replicas are sharded.
+5. Client expects the MicroService to be highly available,
```

Comment:
nit: Change to "the microservice is highly available, tolerating less than half the consensus replicas failing (as ConsensusRSM automatically, reliably and consistently replicates all method calls to all replicas of each shard)"

Reply:
Updated.